config.template.yaml

```diff
@@ -4,13 +4,40 @@
 # ACCESS_TOKEN_SECRET_KEY should contain a random string with enough entropy (at least 32 bytes long) to securely sign all access_tokens for OAuth and Openid connect
 ACCESS_TOKEN_SECRET_KEY: ""
 # RSA_PRIVATE_PEM_STRING should be a string containing the PEM certificate of a private RSA key. It will be used to sign id_tokens for Openid connect authentication
-# In the pem certificates newlines can be replaced by `\n`
-RSA_PRIVATE_PEM_STRING: ""
+# The example below was generated using a 2048-bit RSA key generator
+RSA_PRIVATE_PEM_STRING: |
```

Comment:
I'm not fundamentally against providing a default PEM string for local development, but I am afraid some people may keep it for services exposed over internet